web/src/components/DandiFooter.vue

```diff
@@ -15,7 +15,7 @@
       </cookie-law>
       <v-row>
         <v-col offset="2">
-          &copy; 2019 - present The DANDI Team<br>
+          &copy; 2022 - present The DANDI Team<br>
```

Comment:
ok, I think it should be 

```suggestion
          &copy; 2019 - 2022 The DANDI Team<br>
```

as programmatically set by a bot @yarikoptic matching it to the year of the last change. The year when we forget to update it would mean we are no longer actively programmatically maintaining the project ;)